src/app/contexts/RequestContext/index.tsx

```diff
@@ -26,7 +26,7 @@ export type RequestContextProps = {
   isLite: boolean;
   isNextJs: boolean;
   isUK: boolean;
-  mvtExperiments?: MvtExperiment[] | null;
+  serverSideExperiments?: MvtExperiment[] | null;
```

Comment:
Should we also consider renaming the `MvtExperiment` type to `ServerSideExperiment`?